### PR TITLE
Update dart-2 page for Flutter and --preview-dart-2

### LINKED ISSUE #593: The Dart 2 page should refer to the CHANGELOG file for more details of what is in Dart 2
We should make Dart 2 also refer to the 
https://github.com/dart-lang/sdk/blob/master/CHANGELOG.md#200 file for more details of the contents of Dart 2.

### PR DESCRIPTION
**Pull request by kwalrath**
Fixes #593 by linking to the SDK CHANGELOG.

Contributes to #594.

Staged at https://kw-www-dartlang-1.firebaseapp.com/dart-2

### CODE REVIEW

File: src/dart-2.md [hunk 1]
```diff
@@ -2,23 +2,25 @@
...
-**Don't rely on Dart 2 until it's stable.**
-Dart 2 pre-releases will have a series of incompatible changes.
-Although we appreciate your willingness to prepare for and test Dart 2,
-please don't bet your livelihood on a pre-release.
-For information on potential changes, see the
+**Dart 2 is still changing.**
+For information about what to expect, see the
 [Dart 2 changes](/guides/language/spec#dart-2-changes) section of the
 [Dart Language Specification](/guides/language/spec) page.
+For a complete list of committed (but not necessarily released)
```
Comment (chalin) [inline src/dart-2.md]: 👍

File: src/dart-2.md [hunk 2]
```diff
@@ -27,11 +29,18 @@ For information on potential changes, see the
...
 
-<aside class="alert alert-info" markdown="1">
-**Flutter note:** You don't need to do anything to prepare Flutter code
-for Dart 2. If changes become necessary,
-we'll add Flutter details to this page.
-</aside>
+
+## Migration tips: Flutter
+
+Use `--preview-dart-2` with the `flutter` tool to enable the latest
```
Comment (chalin) [inline src/dart-2.md]: Can we confirmed that the flag name will be `--preview-dart-2` across all platforms?
(It currently seems to be `--preview_dart_2` for Dart VM.)

File: src/dart-2.md [hunk 3]
```diff
@@ -2,23 +2,25 @@
...
+and how you can migrate your code from Dart 1.x and
+older 2.0.0-dev releases.
 
 <aside class="alert alert-warning" markdown="1">
-**Don't rely on Dart 2 until it's stable.**
-Dart 2 pre-releases will have a series of incompatible changes.
-Although we appreciate your willingness to prepare for and test Dart 2,
-please don't bet your livelihood on a pre-release.
-For information on potential changes, see the
+**Dart 2 is still changing.**
```
Comment (anders-sandholm) [inline src/dart-2.md]: "still changing" sounds a bit too scary IMO. What Dart 2 is is not changing. Maybe "Dart 2 is still landing"?

File: src/dart-2.md (hunk 2 above)
Comment (kwalrath) [reply src/dart-2.md]: From all the emails I've seen, the flag name is to be standardized as `--preview-dart-2`.

/cc @kevmoo

**Pull request merged**